blueprints/empress-blog/files/__base__/author/ghost.md

```diff
@@ -7,5 +7,6 @@ website: https://github.com/empress/empress-blog
 twitter:
 facebook:
 location:
+meta: Ghost Meta
```

Comment:
I would prefer this meta example to be in the dummy app files rather than in the generator files, it would be odd to have this in all authors that are generated using `ember g author name` 🤔  could you move it to there for the example?

Reply:
That's not what this does.

The file that gets triggered by `ember g author name` is this one:

https://github.com/empress/empress-blog/pull/103/files#diff-20986730db2cda485bee6fc484ae9f526eb3a679b42cd9c0c29aa4679d8ef9c0

This file you've commented on is the blueprint for installing `empress-blog` itself, which generates the same example files as the dummy app. The way you have this constructed, the changes need to be made in the dummy app *and* here, otherwise the content of the dummy app diverges from what gets generated when `empress-blog` is installed.